reference/src/layout/unions.md

```diff
@@ -60,12 +105,6 @@ assert_eq!(size_of::<U>(), 2);
 # }
 ```
 
-> **NOTE**: U is larger than its largest field, and has therefore 1 byte of
-> trailing padding.
-
 This handling of zero-sized types is equivalent to the handling of zero-sized
 types in struct fields, and matches the behavior of GCC and Clang for unions in
 C when zero-sized types are allowed via their language extensions.
```

Comment:
For structs we called out a C++ compat hazard, does the same apply here?

Reply:
Yes the same applies here. I'll call that out as well.